examples/freertos/Main.cpp

```diff
@@ -0,0 +1,68 @@
+#include "generated/stm32fxxx/PinoutTableDefault.hpp"
+#include "hal_st/instantiations/NucleoUi.hpp"
+#include "hal_st/instantiations/StmEventInfrastructure.hpp"
+#include "osal/Osal.hpp"
+#include "osal/freertos/LowPowerStrategyFreeRtos.hpp"
+#include "osal/freertos_system_time/TimerServiceFreeRtos.hpp"
+#include "services/util/DebugLed.hpp"
+#include <chrono>
+#include <thread>
+
+extern "C" void Default_Handler()
```

Comment:
Perhaps a silly idea. But this Default_Handler is used in many projects, in the same way.

Would it make sense to move this Default_Handler to a separate library, just like is done for the DefaultLinkerScripts and or DefaultClockInit libraries that we provide a  DefaultInterruptHandler library with just this function?

Reply:
Yes! That's certainly something that still needs to be done, but not in this pull request :)